src/nvim/normal.c

```diff
@@ -6593,6 +6594,11 @@ static void nv_open(cmdarg_T *cap)
   }
 }
 
+static void nv_paste(cmdarg_T *cap)
```

Comment:
can we give a hint that mentions `<PasteStart>`. 

```suggestion
/// Handles <PasteStart>, repeats pasted text.
static void nv_paste(cmdarg_T *cap)
```